BParkingNano/plugins/MuonTriggerSelector.cc

```diff
@@ -184,44 +181,52 @@ void MuonTriggerSelector::produce(edm::Event& iEvent, const edm::EventSetup& iSe
       }
 
       //save reco muon 
-      //since we do not store full muon collection => useless to save original muon index
       if(recoMuonMatching_index != -1){
 	pat::Muon recoTriggerMuonCand (muon);
 	recoTriggerMuonCand.addUserInt("trgMuonIndex", trgMuonMatching_index);
 	trgmuons_out->emplace_back(recoTriggerMuonCand);
+
+	//keep track of original muon index for SelectedMuons collection
+	isTriggerMuon[iMuo] = 1;
```

Comment:
Should we have a counter instead of a boolean? This would allow to identify the trigger muon in case we want to have a tag side a probe one. In that case the vector should be initialised to -1 or to muons->size()

Reply:
I realise that isTriggerMuon is previously used as a bool, while here as  std::vector<int> isTriggerMuon(muons->size(), 0);
It's safe but very confusing, I'll change the name